specificatie/resolved/openapi.yaml

```diff
@@ -0,0 +1,467 @@
+openapi: 3.1.0
+info:
+  title: Haal Centraal BRP bewoning
```

Comment:
Moet dit niet BRP API bewoning zijn?

Reply:
Kan inderdaad beter aangepast worden naar:

```suggestion
  title: BRP API bewoning
```

Deze specificatie is ondertussen al verwijderd uit haal-centraal-brp-bewoning en hoeft daar dus niet meer worden aangepast